### ISSUE TITLE
Server thread dump

### ISSUE THREAD

**Issue by Kbryst** [lishid/Orebfuscator#227]
https://pastebin.com/6WQ8EDn4

Setting this up on 1.13.2
Paper Spigot - 597 (currently latest version)

I'm just doing this on a small test server off my pc before moving it to an actual production server. Just wanted to know if this is an error because this is just a small test server, or can this be something that may show up on my main server?

**Comment by ProgrammerDan** [comment]
First pass guess, paper enables an async chunk loader by default. This is
NOT supported by orebfuscator (will cause endless loading loops), so, to
use orebfuscator with Paper, you must turn off the async chunk cache / load
in paper.

**Comment by Kbryst** [comment]
I went ahead and to changed this in the paper.yml:
  async-chunks:
    enable: false
    generation: false
    thread-per-world-generation: false
    load-threads: -1

I am still able to recreate the error

**Comment by ProgrammerDan** [comment]
Might be something new, then.

Describe the process you follow to generate the error? Perhaps share some
more of the crash logs, surrounding entries?

**Comment by Kbryst** [comment]
Sure thing.

So I'm just on a test server, I noticed a first join it popped up (only on the first join).

I'm also testing an /rtp plugin so every time I use it the error shows up again. It looks like its just happening in new chunks to the plugin. 

First join log:
https://pastebin.com/E6CVT6N2

Orebfuscator config:
https://pastebin.com/mUvWVN4V
*Its mostly the default config. 

Testing with using /rtp:
https://pastebin.com/7D8v9GVD




Edit: First join was not on the latest paper spigot, but the testing with /rtp is on the latest paper spigot.

**Comment by ProgrammerDan** [comment]
Alright, thanks for these additional details. Not sure exactly why what I'm
seeing is happening yet.

**Comment by Kbryst** [comment]
Yeah I have no idea either. My first thought was maybe it was 1.12 map rolling to 1.13, but I realize one of the maps that I was on and testing was a brand new map made today.

**Comment by ProgrammerDan** [comment]
Yeah, these results look like from a new map (lots of chunk generation).
Probably related to that. I need to review some recent-ish feature adds
because something is deeply awry here.
